scripts/publish_example_server_container.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+[ -z "$CI" ] && echo "This script is meant to run only from CircleCI." && exit 1;
+[ -z "$DOCKERHUB_USERNAME" ] && echo "Undefined DOCKERHUB_USERNAME, skipping publish" && exit 0;
```

Comment:
This should be `exit 1` IMO. If for some crazy reason this env var was lost in our CI, the build would still pass and that's not a good thing. It's better to know about these kinds of failures.

Reply:
ok